.run/Run all acceptance tests.run.xml

```diff
@@ -0,0 +1,19 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run all acceptance tests" type="GoTestRunConfiguration" factoryName="Go Test">
+    <module name="terraform-provider-axual" />
+    <working_directory value="$PROJECT_DIR$/internal/tests" />
+    <envs>
+      <env name="AXUAL_PASSWORD" value="jr8E@xpGAdJ3k&amp;3&amp;" />
```

Comment:
You seem to have pushed the password again

Reply:
Removed password and username